src/theory/inference_id.cpp

```diff
@@ -411,6 +411,10 @@ const char* toString(InferenceId i)
       return "STRINGS_ARRAY_NTH_EXTRACT";
     case InferenceId::STRINGS_ARRAY_NTH_UPDATE:
       return "STRINGS_ARRAY_NTH_UPDATE";
+    case InferenceId::STRINGS_ARRAY_NTH_UPDATE_SHORTCUT1:
+      return "STRINGS_ARRAY_NTH_UPDATE_SHORTCUT1";
+    case InferenceId::STRINGS_ARRAY_NTH_UPDATE_SHORTCUT2:
+      return "STRINGS_ARRAY_NTH_UPDATE_SHORTCUT2";
```

Comment:
Rather than calling the inferences shortcut 1 and shortcut 2, can we give a name that is more informative?

Reply:
updated.